StringUtils.cpp

```diff
@@ -25,14 +25,12 @@ std::string remQuotes(const std::string& in)
 
 std::string addQuotes(const std::string& in)
 {
-	const auto length = in.size();
-
-	if (length == 0)
+	if (in.empty())
 	{
 		return "\"\"";
```

Comment:
Is there a need for separating this case from the final return?

Reply:
I'd advocate for deleting. If there were trouble, the tests would reveal it.